src/vs/workbench/parts/comments/electron-browser/commentsEditorContribution.ts

```diff
@@ -17,7 +17,7 @@ import { EmbeddedCodeEditorWidget } from 'vs/editor/browser/widget/embeddedCodeE
 import { IEditorContribution } from 'vs/editor/common/editorCommon';
 import { IRange } from 'vs/editor/common/core/range';
 import * as modes from 'vs/editor/common/modes';
-import { peekViewEditorBackground, peekViewResultsBackground, peekViewResultsSelectionBackground } from 'vs/editor/contrib/referenceSearch/referencesWidget';
+import { peekViewEditorBackground, peekViewResultsBackground, peekViewResultsSelectionBackground, peekViewTitleBackground } from 'vs/editor/contrib/referenceSearch/referencesWidget';
```

Comment:
Looks like `peekViewEditorBackground` is unused

Reply:
Removed